en/02_Developer_Guides/01_Templates/02_Common_Variables.md

```diff
@@ -123,18 +123,18 @@ on a per-page basis.
 If you don’t want to include the title tag use `$MetaTags(false)`.
 [/notice]
 
-By default `$MetaTags` renders (assuming 4.11.0 is the current version of silverstripe/framework):
+By default `$MetaTags` renders (assuming 5.1.0 is the current version of `silverstripe/framework`):
```

Comment:
I wonder how many people will think there's 5.1 version :-D

Reply:
There will be before too long :p